src/Header/Header.css

```diff
@@ -0,0 +1,33 @@
+.header {
+  background: linear-gradient(90deg, #672280 1.18%, #A626D3 100%);
+  display: flex;
+  justify-content: space-around;
+  align-items: center;
+
+}
+
+.header-left {
+  display: flex;
+  justify-content: center;
+  gap: 10px;
+}
+
+.header-logo {
+  width: 50px;
+  height: 45px;
+  align-self: center;
+}
+
+.header-left-text {
+  font-family: 'Karla', sans-serif;
```

Comment:
- [ ]  - I think you need to add the `Karla` font. to the project

Reply:
Added